### DISCUSSION TITLE
posts/Go/go-kit-tutorial-06/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#291] [Announcements]
https://www.liwenzhou.com/posts/Go/go-kit-tutorial-06/ 

 李文周的Blog提供免费的全套Go语言学习教程。本文主要介绍了go-kit的教程的第六篇，介绍了如何使用go-kit如何实现服务发现和负载均衡，并以consul为例演示了示例。

**Comment by gotosre** [comment]
mark!
